### ISSUE TITLE
Types missing

### ISSUE THREAD

**Issue by starsoccer** [sebastianwessel/quickjs#6]
Seems like tpyes arent being exported properly or perhaps Im missing something when I install this and try to use it in a typescript project I get the following:

```Could not find a declaration file for module '@sebastianwessel/quickjs'. 'SOMEPATH/node_modules/@sebastianwessel/quickjs/dist/commonjs/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/sebastianwessel__quickjs` if it exists or add a new declaration (.d.ts) file containing `declare module '@sebastianwessel/quickjs';`ts(7016)```

**Comment by sebastianwessel** [comment]
There should be no `@types/sebastianwessel__quickjs` needed.
The package should contain the types already

**Comment by starsoccer** [comment]
Yes Im aware, but from installing this package Im being told that no types are found. Thats just the standard error if no types are found in a package so for whatever reason it seems like they arent being exported properly by default

**Comment by sebastianwessel** [comment]
You're right! Thanks for your feedback 🙏

There was an issue in the configuration. I have published a fixed version, 1.1.1, which should resolve the issue. There should be no additional type import necessary now.

**Comment by starsoccer** [comment]
Thanks that fixed it
